core/app/models/workarea/configuration/params.rb

```diff
@@ -17,12 +17,7 @@ def to_h
               value
             end
 
-          memo[field.key] =
-            if formatted_value.blank? && !field.allow_blank?
-              field.default
-            else
-              formatted_value
-            end
+          memo[field.key] = formatted_value.presence || field.default
```

Comment:
so if someone defines a field

```
field 'Foo',
  type: String,
  default: 'bar',
  allow_blank: true
```
and the admin blanks out the field, the value will be `foo`?

Reply:
Isn't that what a default means?

Reply:
default is the initial value, a user blanking out a field that is allowed to be blank seems intentional enough that it should be left blank. If the field required a value than setting default when left blank would make sense but not in the same of a field that the developer defined as ok to be blank.